src/main/java/mate/academy/spring/service/RepeatedPasswordValidator.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.service;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class RepeatedPasswordValidator
+        implements ConstraintValidator<RepeatedPassword, UserRequestDto> {
+    private String password;
+    private String repeatedPassword;

```

Comment:
We have just one method. In my opinion, you can create local variables.

Reply:
I don't need both of them here as I use DTO with all the necessary date